helm/cluster/templates/clusterapi/_helpers_files.tpl

```diff
@@ -2,9 +2,7 @@
 {{- include "cluster.internal.kubeadm.files.sysctl" $ }}
 {{- include "cluster.internal.kubeadm.files.selinux" $ }}
 {{- include "cluster.internal.kubeadm.files.systemd" $ }}
-{{- include "cluster.internal.kubeadm.files.cgroupv1" $ }}
```

Comment:
All the removed lines in this file were moved to the workers helpers file , because now it's only used for worker nodes.